addons/pos_loyalty/static/src/tours/PosLoyaltyTour.js

```diff
@@ -359,3 +359,26 @@ ProductScreen.check.selectedOrderlineHas('Product A', '2.00', '40.00');
 PosLoyalty.check.orderTotalIs('66.00');
 
 Tour.register('PosLoyaltyMinAmountAndSpecificProductTour', {test: true, url: '/pos/web'}, getSteps());
+
+function createOrderCoupon(totalAmount, couponName, couponAmount) {
+    return [
+        ProductScreen.do.confirmOpeningPopup(),
+        ProductScreen.do.clickHomeCategory(),
+        ProductScreen.do.clickPartnerButton(),
+        ProductScreen.do.clickCustomer("partner_a"),
+        ProductScreen.exec.addOrderline("product_a", "1"),
+        ProductScreen.exec.addOrderline("product_b", "1"),
+        PosLoyalty.do.enterCode("promocode"),
+        PosLoyalty.check.hasRewardLine(`${couponName}`, `${couponAmount}`),
+        PosLoyalty.check.orderTotalIs(`${totalAmount}`),
```

Comment:
maybe add a step to check the point awarded in the PoS. Same for the other tours